### DISCUSSION TITLE
I want to bind the hotkey “Fn+F4” to switch GPU mode. What should I do, please?

### DISCUSSION THREAD

**Discussion by Waximell** [seerge/g-helper#1321] [Q&A] [answered]
As the title, I have no idea how to fill the "action" text field to switch GPU mode, I need some help, thank you~
<img width="699" alt="image" src="https://github.com/seerge/g-helper/assets/59529964/81f73e50-4875-4a67-837b-a47ffdd3a59b">

**Comment by seerge** [comment, marked as answer]
@Waximell hello, you can't :) GPU mode switching is quite invasive process, as it physically turns off dGPU. And it's needed only to save battery (and Optimized mode does that automatically already)

Why do you want to change it so often ?

**Reply by Waximell** [reply]
Well, it doesn't necessarily need to be changed so frequently, actually. I just hope I can switch it more conveniently whether it‘s charging, like turning on the GPU only when playing games and turning it off when not in use, so my computer might be quieter during normal times. Since that is invasive, it seems inappropriate to add this feature. Thank you for your response~
